app/services/food_service.rb

```diff
@@ -0,0 +1,19 @@
+class FoodService
+  def food_search(food)
+    response = conn.get('/fdc/v1/foods/search') do |find|
+      find.params['query'] = "ingredients:#{food}"
+      find.params['pageSize'] = 10
```

Comment:
Great job setting these query params in a block, using a connection object, and making a separate parse_json method.